src/openms/source/FORMAT/FileHandler.cpp

```diff
@@ -238,7 +238,7 @@ namespace OpenMS
       return FileTypes::MZML;
 
     //"analysisXML" aka. mzid (all lines)
-    if (all_simple.hasSubstring("<mzIdentML"))
+    if (all_simple.hasSubstring("<MzIdentML"))
```

Comment:
I had that in mind as well, but technically, XML is case sensitive and any other term is not valid XML -- technically.